pages/index.js

```diff
@@ -11,4 +12,4 @@ function Home() {
   );
 }
 
-export default withReduxSaga(Home, { type: 'GET_REPO_SERVER' });
+export default withReduxSaga(configureStore, Home, { type: 'GET_REPO' });
```

Comment:
So something like:
`withReduxSaga(configureStore)(Home, { type: 'GET_REPO' });` ?

That is absolutely doable!